src/main/java/team/gif/robot/commands/collector/CollectorDown.java

```diff
@@ -0,0 +1,27 @@
+package team.gif.robot.commands.collector;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import team.gif.robot.Robot;
+
+public class CollectorDown extends CommandBase {
+
+    public CollectorDown() {
+        //addRequirements(intake);
+    }
+
+    @Override
+    public void initialize() {
+        Robot.collector.setSolenoid(true);
```

Comment:
This should really be abstracted
```suggestion
        Robot.collector.raise();
```